recipes/mimalloc/all/conandata.yml

```diff
@@ -21,28 +27,23 @@ sources:
     url: "https://github.com/microsoft/mimalloc/archive/v1.6.7.tar.gz"
     sha256: "111b718b496f297f128d842880e72e90e33953cf00b45ba0ccd2167e7340ed17"
 patches:
+  "2.0.7":
+    - patch_file: "patches/2.0.7-0001-change-install-paths.patch"
   "2.0.6":
     - patch_file: "patches/2.0.6-0001-change-install-paths.patch"
-      base_path: "source_subfolder"
   "2.0.5":
     - patch_file: "patches/2.0.6-0001-change-install-paths.patch"
-      base_path: "source_subfolder"
   "2.0.3":
     - patch_file: "patches/2.0.3-0001-change-install-paths-avoid-symlink.patch"
-      base_path: "source_subfolder"
   "2.0.2":
     - patch_file: "patches/2.0.2-0001-change-install-paths-avoid-symlink.patch"
-      base_path: "source_subfolder"
     - patch_file: "patches/2.0.2-0002-include-cstddef-to-get-std-size-t.patch"
-      base_path: "source_subfolder"
+  "1.7.7":
+    - patch_file: "patches/1.7.7-0001-change-install-paths-avoid-symlink.patch"
```

Comment:
Patch fields please!

Reply:
@prince-chrismc 
Thank you for your review!
I added patch_descriptions.